iommi/style_base.py

```diff
@@ -137,7 +137,10 @@
         active_item__tag='span',
         link__tag='a',
         link__attrs__class__iommi_page_link=True,
-        attrs={"data-iommi-page-parameter": lambda paginator, **_: paginator.iommi_path},
+        attrs={
+            "data-iommi-page-parameter": lambda paginator, **_: paginator.iommi_path,
+            'aria-label': 'Pages',
```

Comment:
What is aria-label, and why does it go in base?

Reply:
Accessibility thing.